tartiflette/engine.py

```diff
@@ -256,8 +258,75 @@ def __init__(
             ),
         )
 
+        self._query_executor, self._subcription_executor = self._schema.bake_execute(
+            self._perform_query, self._perform_subsciption
+        )
+
         self._cooked = True
 
+    async def _perform_subsciption(
+        self,
+        schema,
+        document,
+        errors,
```

Comment:
I think that `errors` isn't enough descriptive on what this parameter could contains.

Maybe we could rename it `request_parsing_errors` or something more explicit? (same apply for `_perform_query` method)

What do you think?